pkg/crypto/tinkcrypto/primitive/jwe/ecdhes/register_ecdhes_aead_helper.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package ecdhes
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/google/tink/go/core/registry"
+	gcmpb "github.com/google/tink/proto/aes_gcm_go_proto"
+	tinkpb "github.com/google/tink/proto/tink_go_proto"
+	"github.com/square/go-jose/v3"
+
+	"github.com/hyperledger/aries-framework-go/pkg/crypto/tinkcrypto/primitive/jwe/subtle/ecdhes"
+)
+
+const (
+	// nolint:gochecknoglobals
+	aesGCMTypeURL = "type.googleapis.com/google.crypto.tink.AesGcmKey"
+)
+
+// registerECDHESEncHelper registers a DEM helper.
+type registerECDHESEncHelper struct {
+	demKeyURL        string
+	keyData          []byte
+	symmetricKeySize uint32
+	encAlg           jose.ContentEncryption
+}
+
+var _ ecdhes.EncryptHelper = (*registerECDHESEncHelper)(nil)
+
+// newRegisterECDHESEncHelper initializes and returns a RegisterECDHESDemHelper
+func newRegisterECDHESEncHelper(k *tinkpb.KeyTemplate) (*registerECDHESEncHelper, error) {
+	var keyLen uint32
+
+	var skf []byte
+
+	var encAlg jose.ContentEncryption
+
+	u := k.TypeUrl
+
+	if strings.Compare(u, aesGCMTypeURL) == 0 {
+		gcmKeyFormat := new(gcmpb.AesGcmKeyFormat)
+
+		err := proto.Unmarshal(k.Value, gcmKeyFormat)
+		if err != nil {
+			return nil, err
+		}
+
+		keyLen = gcmKeyFormat.KeySize
+		switch keyLen {
+		case 16: // nolint:gomnd
```

Comment:
instead of nolint, you could create constants.

Reply:
sure, I'll update

Reply:
there's no more hard coded key size in my latest change